docs/user/integration/dynatrace/README.md

```diff
@@ -122,10 +121,10 @@ To start ingesting custom spans and Istio spans, you must enable the Istio traci
     The default configuration has the **randomSamplingPercentage** property set to `1.0`, meaning it samples 1% of all requests. To change the sampling rate, adjust the property to the desired value, up to 100 percent.
 
     > **CAUTION:** Be cautious when you configure the **randomSamplingPercentage**:
-    > - Traces might consume a significant storage volume in Cloud Logging Service.
+    > - Traces might consume a significant storage volume in Dynatrace.
```

Comment:
Is that relevant? Maybe rather refer to Dynatrace pricing. Check if there is a price per span.